contracts/application/versioning/Package.sol

```diff
@@ -5,38 +5,37 @@ import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 
 /**
  * @title Package
- * @dev Complex collection of contracts that groups contracts into versions
- * @dev Contracts with the same name can have different implementation addresses in different versions
+ * @dev Collection of contracts grouped into versions.
+ * Contracts with the same name can have different implementation addresses in different versions.
  */
 contract Package is Ownable {
   /**
-   * @dev This event signals the addition of a version to the package
-   * @dev Version is not indexed due to truffle testing constraints
-   * @param version Name of the version added
-   * @param provider ImplementationProvider associated with the added version
+   * @dev Emitted when a version is added to the package.
+   * XXX The version is not indexed due to truffle testing constraints.
+   * @param version Name of the added version.
+   * @param provider ImplementationProvider associated with the version.
    */
   event VersionAdded(string version, ImplementationProvider provider);
 
   /*
-   * @dev Mapping that stores the association between versions 
-   * @dev and their implementation providers.
+   * @dev Mapping for version and their implementation providers.
```

Comment:
"Mapping between versions and..."?
"Mapping associating versions and..."?
"version" -> "versions" for sure.

Reply:
let's try associating.